### DISCUSSION TITLE
SideNavMenu must have children?

### DISCUSSION THREAD

**Discussion by gcattan** [carbon-design-system/carbon#12981] [Help] [answered]
Hello, 

We made the migration to carbon v11 recently. We noticed that the `SideNavMenu` is now force to have children. This comes from the refactoring [here](https://github.com/carbon-design-system/carbon/commit/4cb3f0e6cce19a298044b26e977950e8fdd52d8b).
It is not clear to me if the change is made by design, or if this is a bug.

Thanks for your reply!

**Comment by tw15egan** [comment]
Just curious, what is the use case for rendering a `SideNavMenu` without children? I would expect it not to be rendered at all if there are no children, rather than rendering an empty menu that is clickable (and will show no items when expanded)

**Comment by tw15egan** [comment, marked as answer]
The use case is as follows (https://github.com/carbon-design-system/carbon/issues/12986): 

![image](https://user-images.githubusercontent.com/11928039/213512979-1f2eca7c-6c7b-4695-91d2-bd1ebdf7dab9.png)


This can be achieved by setting `defaultExpanded={true}` on `SideNavMenu`
